scripts/merge-stream-chat-css-docs.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+usage() {
+  echo "Missing path to stream-chat-css directory"
+  echo "Usage: $(basename $0) <path_to_stream-chat-css_directory>"
+}
+
+main() {
+  if [ $# -eq 0 ]; then
+      usage
+      exit 0
+  fi
+
+  STREAM_CHAT_CSS_PATH=$1;
+  cp -r "$STREAM_CHAT_CSS_PATH"/* ./docusaurus/docs/React;
```

Comment:
Will this merge the assets folder from the React SDK folder and the CSS folder?

Reply:
@szuperaz yes, but for clarity I will rename the var to STREAM_CHAT_CSS_DOCS_PATH.

Reply:
I was asking if it's a problem that both folders had folders named `assets`, but I based on google the `cp` command can handle that